hw/ip_templates/pinmux/rtl/pinmux.sv.tpl

```diff
@@ -492,6 +492,11 @@ module pinmux
     assign mio_to_periph_o[k] = mio_mux[reg2hw.mio_periph_insel[k].q];
   end
 
+  if (AlignedMuxSize > 2**$clog2(NMioPads + 2)) begin : gen_align_unused
+    logic unused_mio_mux_signals;
+    assign unused_mio_mux_signals = ^{mio_mux[AlignedMuxSize-1:2**$clog2(NMioPads + 2)]};
```

Comment:
Only the lowest `NMioPads + 2` bit of `mio_mux` are used, right? So shouldn't this be
```suggestion
    assign unused_mio_mux_signals = ^{mio_mux[(AlignedMuxSize - 1):(NMioPads + 2)]};
```
?

The same question also applies to `mio_wkup_mux` and `dio_wkup_mux` below